src/views/Player.vue

```diff
@@ -16,25 +17,48 @@
         :questions="questions"
         v-model:currentQuestionIndex="currentQuestionIndex"
         v-model:responses="responses"
+        @submit-question="submitQuestion"
```

Comment:
is this some TS formatting? The kebab case for method handlers? I always wrote them like `@submitQuestion`.